Setup/src/cls/PopulationResponse.cls

```diff
@@ -0,0 +1,7 @@
+/// This class holds population density information retrieved from an API
+Class Demo.PopulationResponse Extends (%Persistent, Ens.Util.MessageBodyMethods)
```

Comment:
```suggestion
Class Demo.PopulationResponse Extends (Ens.Request)
```

Reply:
also, is the Ens.Util.MessageBodyMethods class necessary? (this goes for all of these classes.